Dockerfile

```diff
@@ -1,36 +1,36 @@
 # ------------------------------------------------------------------------------
 # Builder Stage
 # ------------------------------------------------------------------------------
-FROM golang:1.19-bullseye AS build
+FROM golang:1.19-alpine AS build
+
+RUN apk add --no-cache make
 
 WORKDIR /build
-ADD . /build
 
+COPY go.mod go.sum ./
 RUN go mod download
-RUN make build
+
+COPY Makefile main.go ./
+COPY static/ static/
+COPY pkg/ pkg/
+COPY .git/ .
+RUN CGO_ENABLED=0 make build
 
 # ------------------------------------------------------------------------------
-# Release Stage
+# Fetch wait-for
 # ------------------------------------------------------------------------------
-FROM debian:bullseye-slim
-
-RUN \
-  apt-get update && \
-  apt-get install -y ca-certificates openssl netcat curl gnupg lsb-release && \
-  update-ca-certificates
-
-RUN \
-  curl --silent https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add && \
-  echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee  /etc/apt/sources.list.d/pgdg.list && \
-  apt-get update && apt-get install -y postgresql-client
```

Comment:
Pgweb absolutely needs the postgres client package since it utulizes `pg_dump` under the hood for exporting data dumps.

Reply:
Oh, ok, I didn't now that, thanks.